source/includes/inkling2-reference/_concepts.html.md

```diff
@@ -0,0 +1,109 @@
+# Concepts
+
+The `concept` keyword declares an abstract concept that is to be learned by the system. It can be a feature (such as a curvy line in an image) or a goal (such as high score in a game). Ultimately, a concept takes the form of a data transform. By declaring a concept, you are instructing the AI Engine that this is a node in the concept graph that must be learned.
```

Comment:
```suggestion
The `concept` keyword declares an abstract concept that is to be learned by the system. It can be a feature (such as a curvy line in an image) or a goal (such as high score in a game). Ultimately, a concept represents a function from input to output. By declaring a concept, you are instructing the AI Engine that this is a node in the concept graph that must be learned.
```

Once we have non-learned concepts, we'll need to update the last sentence here...